pkgs/tools/video/bento4/default.nix

```diff
@@ -23,17 +23,21 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
     mkdir -p $out/{lib,bin}
-    find -iname '*.so' -exec mv --target-directory="$out/lib" {} \;
+    find -iname '*${stdenv.hostPlatform.extensions.sharedLibrary}' -exec mv --target-directory="$out/lib" {} \;
     find -maxdepth 1 -executable -type f -exec mv --target-directory="$out/bin" {} \;
     runHook postInstall
   '';
 
+  # Patch binaries to use our dylib
+  fixupPhase = lib.optional stdenv.isDarwin ''
```

Comment:
```suggestion
  postInstall = lib.optionalString stdenv.isDarwin ''
```

Reply:
Amended, thanks. Can I ask for some background on why? Very much still learning the ins and outs of nixpkgs.

Reply:
The default phases are [not void](https://nixos.org/manual/nixpkgs/stable/#ssec-fixup-phase), and overriding `fixupPhase` with `""` on Linux is essentially doing `dontFixup = true`.